torch/csrc/jit/dynamic_dag.h

```diff
@@ -265,8 +280,22 @@ EdgeData<T> DynamicDAG<T>::removeVertex(Vertex<T>* v) {
  * We find all children of y in the affected region. deltaF = {y, a, b}
  * We find all parents of x via DFS. deltaB = {c, d, x}
  *
- * Now, we reorder all vertices in deltaB to come before deltaF.
- * result = {c, d, x, y, a, b}, and we assign these {1, 2, 3, 4, 5, 6}.
+ * Now, we reorder all vertices in deltaB to come before deltaF. This is
+ * a little involved and happens in four steps:
+ *
+ * 1) sort all vertices in deltaB, and all vertices in deltaF.
+ * deltaB (sorted) = {c(2), d(4), x(6)}. deltaB ords = { 2, 4, 6 }
+ * deltaF (sorted) = {y(1), a(3), b(5)}. deltaF ords = { 1, 3, 5 }
+ *
+ * 2) append the two lists: the result is the order we want these vertices to have.
+ * L = {c(2), d(4), x(6), y(1), a(3), b(5)}.
+ *
+ * 3) Merge the sorted ords: R = { 1, 2, 3, 4, 5, 6 }.
+ *
+ * 4) Reassign the vertices with the sorted ords.
```

Comment:
The reassignment details are important too! Just add a sentence saying that we always use vertices of deltaB, then x, then y and finally deltaF.